api/src/main/java/org/openmrs/module/openconceptlab/importer/Saver.java

```diff
@@ -88,6 +88,12 @@ public Item saveConcept(final CacheService cacheService, final Import anImport,
 		if (item != null && item.getVersionUrl().equals(oclConcept.getVersionUrl())) {
 			return new Item(thisImport, oclConcept, ItemState.UP_TO_DATE);
 		}
+		
+		if (oclConcept.getSource() != null && oclConcept.getId() != null) {
+			if (cacheService.getConceptByMapping(oclConcept.getSource(), oclConcept.getId()) != null) {
+				return new Item(thisImport, oclConcept, ItemState.UP_TO_DATE);
+			}
+		}
```

Comment:
Can you explain this block?  I may need to see it better in context, but might this prevent updates from happening?

Reply:
This needs an extra guard, but the idea is this: If I have a concept in my dictionary marked SAME-AS CIEL:1066, then we shouldn't load the concept CIEL:1066 into the concept dictionary. The extra guard is to make sure that that item isn't the same as the item we're importing (based on the OCL URL).

Originally, I had this written to return an `ItemState.DUPLICATE`, but I wasn't sure it was worth the effort to add a new `ItemState` for this.